### ISSUE TITLE
Test Accuracy

### ISSUE THREAD

**Issue by mfaraki** [lyakaap/VAT-pytorch#3]
Hi,
Thanks for sharing. Are the hyper-parameters set optimally? I get around 50% test accuracy on the CIFAR10 dataset, which I assume is very low.

Thanks

**Comment by sndnyang** [comment]
I think the hyper-parameters are not optimally. I got 58% test accuracy on cifar10. Using another code with the same arch CNN(the hyper-parameters are really different), I got around 70%.

**Comment by sndnyang** [comment]
By the way, the arch used is much simpler than the one in the paper.

**Comment by luzai** [comment]
May I ask the optimal hyperparameter? 
I try with `xi=1e-6, eps=8, batch_size=128`, performance is better, the test acc is 60.74%.

**Comment by LesiChou** [comment]
Hi! But I found that If you remove the VAT, you could get a similar results.
